packages/x-scheduler/src/material/internals/components/day-time-grid/DayGridCell.tsx

```diff
@@ -1,20 +1,56 @@
 'use client';
 import * as React from 'react';
+import { useStore } from '@base-ui-components/utils/store';
 import { DayGrid } from '../../../../primitives/day-grid';
 import { useAdapter } from '../../../../primitives/utils/adapter/useAdapter';
 import { isWeekend } from '../../../../primitives/utils/date-utils';
 import { useEventOccurrencesWithDayGridPosition } from '../../../../primitives/use-event-occurrences-with-day-grid-position';
 import { EventPopoverTrigger } from '../event-popover';
 import { DayGridEvent } from '../event';
+import { useEventCalendarStoreContext } from '../../../../primitives/utils/useEventCalendarStoreContext';
+import { selectors } from '../../../../primitives/use-event-calendar';
+
 import './DayTimeGrid.css';
+import { useEventPopover } from '../event-popover/EventPopoverContext';
 
 export function DayGridCell(props: DayGridCellProps) {
   const { day, row } = props;
   const adapter = useAdapter();
   const placeholder = DayGrid.usePlaceholderInDay(day.value, row);
+  const store = useEventCalendarStoreContext();
+  const cellRef = React.useRef<HTMLDivElement | null>(null);
+  const rawPlaceholder = useStore(store, selectors.occurrencePlaceholder);
```

Comment:
This would every cell to render whenever the placeholder moves during a dnd.
I think we can create a new selector that directly returns `isCreation` (`selectors.isCreatingNewEventInDayGridCell` for example)